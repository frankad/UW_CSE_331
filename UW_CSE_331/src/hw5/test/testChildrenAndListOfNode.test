# Sample testing: a case list children of node for three node and three edges:
# It tests to list the children node and edge added to the graph 
# it can be created from three nodes and edge between them can be added.
# Then a lsit of node in the graph and children nodes of each node can be
# printed out.

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2
AddEdge graph1 n2 n3 e3

# Print the nodes in the graph and the outgoing edges from n1, n2
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2