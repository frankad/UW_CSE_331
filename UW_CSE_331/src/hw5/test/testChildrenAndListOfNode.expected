# Sample testing: a case list children of node for three node and three edges:
# It tests to list the children node and edge added to the graph 
# it can be created from three nodes and edge between them can be added.
# Then a lsit of node in the graph and children nodes of each node can be
# printed out.

# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n3 in graph1
added edge e3 from n2 to n3 in graph1

# Print the nodes in the graph and the outgoing edges from n1, n2
graph1 contains: n1 n2 n3
the children of n1 in graph1 are: n2(e1) n3(e2)
the children of n2 in graph1 are: n3(e3)